### DISCUSSION TITLE
InteractiveList item disabled

### DISCUSSION THREAD

**Discussion by welitoonl** [TiendaNube/nimbus-design-system#210] [Q&A]
Olá time, estou com dúvida se é possível que possamos "desabilitar" um InteractiveList, como temos na versão antiga do InteractiveList

Por exemplo, no componente `InteractiveList` de @tiendanube/components, conseguimos passar a prop `disabled` para suas options, por consequência, o visual das opções desabilitadas fica desta forma:
![image](https://github.com/TiendaNube/nimbus-design-system/assets/69065140/ca82f583-1ec6-421a-a9aa-11d750174f95)

Entretanto para [`InteractiveList`](https://nimbus.tiendanube.com/documentation/patterns/interactive-list) de Nimbus só temos a possibilidade de adicionar o disabled dentro do objeto `iconButton` por exemplo, o que por consequência faz com que só o botão fique desabilitado, conforme imagem abaixo: 
<img width="586" alt="image" src="https://github.com/TiendaNube/nimbus-design-system/assets/69065140/b716fcad-1f78-4e52-bb36-14ac8be1847e">

Conforme imagem, o uso é para a restrição de plano freemium nas formas de entrega, onde quando a loja é freemium, não é possível instalar os meios de envio, sendo mostrado no modal, as opções disponíveis caso ele suba de plano.

**Comment by harrytiendanube** [comment]
@welitoonl  de que manera resolvieron este caso?. Desde su lado forzaron a que esté desactivado o se cambio el comportamiento?
